### ISSUE TITLE
Calling rerender() on views does not call render() in 1.13.x

### ISSUE THREAD

**Issue by kanongil** [emberjs/ember.js#11515]
If a view implements custom rendering by specifying a `render` function, it is not called again on `rerender()`.

Failing JSBin demo: http://jsbin.com/kekede/6/edit?html,js,output

**Comment by rwjblue** [comment]
Thank you for opening the issue and for the reproduction.

**Comment by kanongil** [comment]
Anything going on with this? This issue is blocking my 1.13 upgrade.

**Comment by rwjblue** [comment]
I haven't had a chance to dig into it yet, but it definitely seems like a regression for 1.x. Though, `render` will be going away in 2.x we definitely should make sure that this works properly in 1.x...

**Comment by Vincz** [comment]
Any update about this? Many thanks!

**Comment by kanongil** [comment]
I just had a look at the 1.13.6 api docs, and `render` has been obliterated without any deprecation warnings or notices. Weirdly, `rerender` still mentions it.

**Comment by rwjblue** [comment]
@kanongil - Darn! We definititely missed adding that deprecation, sorry about that! 

Having a `render` function in a view or component should absolutely print a deprecation, I'm working on adding the proper deprecation for 1.13.7.

**Comment by karanjthakkar** [comment]
@rwjblue I am on 1.13.7 right now and this is breaking my component functionality. Anything I can do to work around or make it future-proof?

**Comment by kushdilip** [comment]
@karanjthakkar did you find any solution. I just did the upgrade and now all my components are broken. I am using rerender everywhere to switch between readonly and editonly modes.

**Comment by karanjthakkar** [comment]
@kushdilip I used the new `Ember.Helper` syntax and event publishing to `recompute` (rerender) the DOM.

**Comment by kushdilip** [comment]
@karanjthakkar  Can you paste a snippet or something on how did you implement the helper. I didn't find any event in Component code which triggers DOM rerender.

**Comment by kenips** [comment]
ping @rwjblue - https://github.com/rwjblue/ember.js/tree/call-legacy-render-on-rerender going to be make it?

**Comment by dmuneras** [comment]
Hi,

I am facing the same problem with an addon I am using in my application. I implemented a way-around (probably a bit dirty), but it works. I used the `willRender` hook  [documentation](http://emberjs.com/blog/2015/06/12/ember-1-13-0-released.html), which is called after a rerender. I renamed the overrided `render(buffer)` function of that addon. The buffer attribute is a "local buffer" and I am rendering it in the tempate using `{{{ mylocalBuffer }}}`.

I just tested and it solved my problem, could be a better solution but it works for the moment.

Daniel.
